Help Desk/helpDesk_editDepartment.php

```diff
@@ -16,15 +16,44 @@
 You should have received a copy of the GNU General Public License
 along with this program. If not, see <http://www.gnu.org/licenses/>.
 */
-
-
+use Gibbon\Forms\Form;
+use Gibbon\Module\HelpDesk\Domain\DepartmentGateway;
 $page->breadcrumbs->add(__('Edit a Department'));
 
-if (!isActionAccessible($guid, $connection2, '/modules/Help Desk/helpDesk_editDepartment.php')) {
+if (!isActionAccessible($guid, $connection2, '/modules/Help Desk/helpDesk_manageDepartments.php')) {
     //Acess denied
     $page->addError(__('You do not have access to this action.'));
 } else {
     //Proceed!
+    $departmentID = $_GET['departmentID'] ?? '';
+
+    $departmentGateway = $container->get(DepartmentGateway::class);
+    $values = $departmentGateway->getByID($departmentID);
+    
+    $form = Form::create('createDepartment',  $gibbon->session->get('absoluteURL') . '/modules/' . $gibbon->session->get('module') . '/helpDesk_editDepartmentProcess.php', 'post');
+    $form->addHiddenValue('address', $gibbon->session->get('address'));
+
+    $row = $form->addRow();
+        $row->addLabel('departmentName', __('Department Name'));
+        $row->addTextField('departmentName')
+            ->readOnly()
```

Comment:
Why is this read only?

Reply:
If its to keep the name unique, I can handle that.

Reply:
Yeah it's to keep the name unique, I'll let you handle it :)